scheduler/plugin/host.go

```diff
@@ -186,47 +192,82 @@ func paramsFromContext(ctx context.Context) *stack {
 	return ctx.Value(stackKey{}).(*stack)
 }
 
-func k8sApiNodeFn(ctx context.Context, mod wazeroapi.Module, stack []uint64) {
+func (h host) k8sApiNodeFn(ctx context.Context, mod wazeroapi.Module, stack []uint64) {
+	nodename := uint32(stack[0])
+	nodenameLen := uint32(stack[1])
+	buf := uint32(stack[2])
+	bufLimit := bufLimit(stack[3])
+
+	var nodeName string
+	if b, ok := mod.Memory().Read(nodename, nodenameLen); !ok {
```

Comment:
Sorry, but at first, I couldn't understand this code.
This code retrieves the node name from the wasm module and returns nodeinfo to the wasm module, right?
Additionally, I couldn't understand why this function is necessary even though k8sSchedulerCurrentNodeNameFn exists. (I understand it now)

Could you leave a brief comment explaining this function?

Reply:
Added the comment though, what's unclear?

Reply:
Lgtm 😀